src/components/display/Alert.jsx

```diff
@@ -0,0 +1,55 @@
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import {Box, Snackbar} from '@material-ui/core';
+import {Alert as MuiAlert} from '@material-ui/lab';
+
+/**
+ * Alert component
+ */
+const Alert = ({id = 'alert', message = null}) => {
+  const [localMessage, setLocalMessage] = useState(message);
+
+  useEffect(() => {
+    if (localMessage !== message) {
+      setLocalMessage(message);
+    }
+  }, [message]);
+
+  const handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
+      setLocalMessage(null);
+    }
+  };
+
+  if (localMessage) {
+    return (
+      <Box id={id}>
+        <Snackbar
+          key={localMessage}
+          open={Boolean(localMessage)}
+          onClose={handleClose}
+          anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'right'
+          }}
+        >
+          <MuiAlert severity="error" variant="filled">
```

Comment:
💡 

What about being able to change the type of alert (error, warning, info, success)?